etcd/getting-started-with-etcd.md

```diff
@@ -6,6 +6,36 @@ Application containers running on your cluster can read and write data into etcd
 
 <a class="btn btn-default" href="../latest/api.html">Complete etcd API Docs</a>
 
+## Setting up etcd
+
+Container Linux comes pre-installed with an `etcd-wrapper` script which can be configured to **run etcd from inside of a container**. Container Linux also comes pre-installed with an `etcd-member.service` systemd unit which by default uses the etcd3 image. This means you to use etcd3 without needing to install the etcd3 binary.
```

Comment:
it isn't configured to run etcd from inside of a container; it only does that.

Please say something like: "Container Linux includes an `etcd-member.service` systemd unit which by runs etcd v3.0+ via a container image. This means you can use etcd v3.0+ on Container Linux without installing a binary."